Google.Api.Gax.Grpc.Tests/Rest/PartialDecodingStreamReaderTest.cs

```diff
@@ -0,0 +1,165 @@
+﻿/*
+ * Copyright 2022 Google LLC
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Google.Api.Gax.Grpc.Rest.Tests
+{
+    public class PartialDecodingStreamReaderTest
+    {
+        private static readonly string DataStr;
+        static PartialDecodingStreamReaderTest()
+        {
+            var foo = new JObject
+            {
+                ["foo"] = 1
+            };
+
+            var bar = new JObject
+            {
+                ["bar"] = 2
+            };
+
+            DataStr = JsonConvert.SerializeObject( new [] { foo, bar }, Formatting.Indented);
+        }
+
+        /// <summary>
+        /// Test coarse split data.
+        /// </summary>
+        [Fact]
+        public async void DecodingStreamReaderTestByLine()
+        {
+            
+            StreamReader reader = new MyStreamReader(DataStr.Split('\n'));
+            var decodingReader = new PartialDecodingStreamReader<JObject>(Task.FromResult(reader), JObject.Parse);
+
+            var result = await decodingReader.MoveNext(CancellationToken.None).ConfigureAwait(false);
```

Comment:
There's no need for `ConfigureAwait` in unit tests.

Reply:
👍